.eslintrc.js

```diff
@@ -76,6 +72,15 @@ module.exports = {
         "@typescript-eslint/no-explicit-any": "off",
         "jest/no-standalone-expect": "off"
       }
+    },
+    {
+      extends: ["plugin:perfectionist/recommended-natural-legacy"],
+      files: "packages/+(api)/**/*",
```

Comment:
Just an understanding question: what does this the + and () mean in `+(api)` mean? Is it "only in the api directory" and then when we do the other packages it'll be like `+(api|core)` or something?

Reply:
https://www.digitalocean.com/community/tools/glob

Reply:
+ is the one or more matcher

Reply:
For this PR I could have just used `packages/api/**/*` but I was messing with that to make sure the glob pattern was picking up the right packages so thought I'd leave it in as I'm going to be adding to it shortly anyway :D

Reply:
Okayyy, that makes sense - thanks (and also for the glob tool, didn't know that was a thing)!